tests/tests_ptp_multi.yml

```diff
@@ -42,23 +42,25 @@
 
             - name: Run pmc
               # yamllint disable-line rule:line-length
-              command: pmc -u -b 0 -d 0 -s /var/run/timemaster/ptp4l.0.socket 'GET DOMAIN'
+              command: pmc -u -b 0 -d 0 -s /var/run/timemaster/ptp4l.0.socket 'GET DOMAIN' 'GET PORT_DATA_SET'
               register: pmc
 
-            - name: Check PTP domain
+            - name: Check PTP domain and port state
               assert:
                 that:
                   - "'domainNumber 0' in pmc.stdout"
+                  - "'portState               LISTENING' in pmc.stdout"
 
             - name: Run pmc
               # yamllint disable-line rule:line-length
-              command: pmc -u -b 0 -d 1 -s /var/run/timemaster/ptp4l.1.socket 'GET DOMAIN'
+              command: pmc -u -b 0 -d 1 -s /var/run/timemaster/ptp4l.1.socket 'GET DOMAIN' 'GET PORT_DATA_SET'
               register: pmc
 
-            - name: Check PTP domain
+            - name: Check PTP domain and port state
               assert:
                 that:
                   - "'domainNumber 1' in pmc.stdout"
+                  - "'portState               LISTENING' in pmc.stdout"
```

Comment:
it seems fragile to require an exact amount of whitespace here - I suggest something like this:
```
 - pmc.stdout is search("portState\s+LISTENING")
```